tests/crowdsourcing/tasks/test_acute_eval.py

```diff
@@ -0,0 +1,361 @@
+            )
+            relative_config_path = os.path.join(relative_task_directory, 'conf')
+            with initialize(config_path=relative_config_path):
+                self.config = compose(
+                    config_name="example",
+                    overrides=[
+                        f'+mephisto.blueprint._blueprint_type={AcuteEvalBlueprint.BLUEPRINT_TYPE}',
+                        f'+mephisto/architect=mock',
+                        f'+mephisto/provider=mock',
+                        f'mephisto.blueprint.block_on_onboarding_fail={False}',
+                        f'+task_dir={TASK_DIRECTORY}',
+                        f'+current_time={int(time.time())}',
+                    ],
+                )
+                # TODO: when Hydra 1.1 is released with support for recursive defaults,
+                #  don't manually specify all missing blueprint args anymore, but
+                #  instead define the blueprint in the defaults list directly.
+                #  Currently, the blueprint can't be set in the defaults list without
+                #  overriding params in the YAML file, as documented at
+                #  https://github.com/facebookresearch/hydra/issues/326 and as fixed in
+                #  https://github.com/facebookresearch/hydra/pull/1044.
+
+            self.data_dir = tempfile.mkdtemp()
+            database_path = os.path.join(self.data_dir, "mephisto.db")
+            self.db = LocalMephistoDB(database_path)
+            self.config = augment_config_from_db(self.config, self.db)
+            self.config.mephisto.architect.should_run_server = True
+            self.operator = Operator(self.db)
+            self.operator.validate_and_run_config(
+                self.config.mephisto, shared_state=None
+            )
+            channel_info = list(self.operator.supervisor.channels.values())[0]
+            server = channel_info.job.architect.server
+
+            # Register a worker
+            mock_worker_name = "MOCK_WORKER"
+            server.register_mock_worker(mock_worker_name)
+            workers = self.db.find_workers(worker_name=mock_worker_name)
+            worker_id = workers[0].db_id
+
+            # Register an agent
+            mock_agent_details = "FAKE_ASSIGNMENT"
+            server.register_mock_agent(worker_id, mock_agent_details)
+            agent = self.db.find_agents()[0]
+            agent_id_1 = agent.db_id
+
+            # Set initial data
+            server.request_init_data(agent_id_1)
+
+            # Make agent act
+            server.send_agent_act(
+                agent_id_1, {"MEPHISTO_is_submit": True, "task_data": DESIRED_OUTPUTS}
+            )
+
+            # Check that the inputs and outputs are as expected
+            agent = self.db.find_agents()[0]
+            state = agent.state.get_data()
+            self.assertEqual(DESIRED_INPUTS, state['inputs'])
+            self.assertEqual(DESIRED_OUTPUTS, state['outputs'])
+
+
+except ImportError:
```

Comment:
sorry for the late review - we usually handle these `ImportError`s via a wrapper, e.g. `@testing_utils.skipUnlessMephisto` --> see https://github.com/facebookresearch/ParlAI/blob/master/parlai/utils/testing.py#L72. Is there a reason we did not do this here?

Reply:
Yeah, good catch - I tried to do that here, but in this case, if I can't import objects from `mephisto`, (say, `LocalMephistoDB`, `Operator`, etc.), I won't be able to define `TestAcuteEval` at all, even with a `@testing_utils.skipUnlessMephisto` decorator, because those unimportable objects are called in the test class. That's not the case with checks like `skipUnlessTorch`, where we don't try to do `import torch` in the test module itself

Reply:
Ahh true - you could always import within the class and set it as an attr:

```
import mephisto
self.mephisto = mephisto
```

which isn't exactly super clean per se, so what you did works for me